src/toolkits/style_transfer/style_transfer.cpp

```diff
@@ -616,23 +615,27 @@ void style_transfer::train(gl_sarray style, gl_sarray content,
 
 std::shared_ptr<MLModelWrapper> style_transfer::export_to_coreml(
     std::string filename, std::map<std::string, flexible_type> opts) {
-  flex_int image_width = read_state<flex_int>("image_width");
-  flex_int image_height = read_state<flex_int>("image_height");
+  const flex_int image_width =
+      read_opts<flex_int>(opts, "image_width", /* delete item */ true);
+  const flex_int image_height =
+      read_opts<flex_int>(opts, "image_height", /* delete item */ true);
+  const flex_int include_flexible_shape =
+      read_opts<flex_int>(opts, "include_flexible_shape", /* delete item */ true);
 
   flex_dict user_defined_metadata = {
       {"model", read_state<flex_string>("model")},
-      {"max_iterations", read_state<flex_int>("max_iterations")},
-      {"training_iterations", read_state<flex_int>("training_iterations")},
+      {"max_iterations", std::to_string(read_state<flex_int>("max_iterations"))},
+      {"training_iterations", std::to_string(read_state<flex_int>("training_iterations"))},
       {"type", "StyleTransfer"},
-      {"content_feature", read_state<flex_string>("content_feature")},
-      {"style_feature", read_state<flex_string>("style_feature")},
-      {"num_styles", read_state<flex_string>("num_styles")},
-      {"version", get_version()},
+      {"content_feature", "image"},
+      {"style_feature", "image"},
```

Comment:
Why are we storing the hard-coded "image" here instead of the column name from the user?

Reply:
Since we used a predefined model_spec to generate the layers, it seemed like it would be difficult to change the input and output layer names at the time. Ehh I'll fix it!

Reply:
Matched behaviors to the original python export to CoreML